sound/soc/intel/boards/Kconfig

```diff
@@ -454,7 +454,7 @@ config SND_SOC_INTEL_SOF_CML_RT1011_RT5682_MACH
         select SND_SOC_RT1011
         select SND_SOC_RT5682
         select SND_SOC_DMIC
-        select SND_SOC_HDAC_HDMI
+        select SND_SOC_HDAC_HDMI if SND_SOC_SOF_HDA_LINK
```

Comment:
this one makes no sense, see line 442, you are already in the case where HDA_LINK is required.

Reply:
@plbossart you are right.